spec/models/financial_change_log_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe FinancialChangeLog, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Something in here just to check the roundtripping of the json data would be great